vital/types/detected_object.h

```diff
@@ -120,6 +141,27 @@ class VITAL_EXPORT detected_object
    */
   void set_bounding_box( const bounding_box_d& bbox );
 
+
+  /**
+   * @brief Get geo_point from this detection.
+   *
+   * The geo_point for this detection is returned. This point is in
+   * world coordinates. A default constructed (invalid) geo_point
+   * is returned if no point has been supplied for this detection.
+   *
+   * @return A copy of the geo_point.
+   */
+  kwiver::vital::geo_point geo_point() const;
+
+  /**
+   * @brief Set new geo_point for this detection.
+   *
+   * The supplied geo_point replaces the point for this detection.
+   *
+   * @param gp geo_point for this detection.
+   */
+  void set_geo_point( const kwiver::vital::geo_point& gp );
```

Comment:
BTW, I see this file was already wrong, but [[types.qualified]](../tree/master/doc/code-style.rst)...

Reply:
I corrected this, note that it would be better to link to an rtd page of the code style, for whatever reason, when I click on this link in github this link resolves to : https://github.com/Kitware/kwiver/compare/tree/master/doc/code-style.rst?expand=1

Which is not helpful at all....

Reply:
Unfortunately, RTD isn't going to help much either; at least not directly. There is no easy, non-repetitive way (AFAICT) to generate anchors for non-headings short of writing custom code. If there was, github would equally have anchors and an RTD version wouldn't be needed. Lacking such custom code, an RTD version doesn't have anchors either.